src/TimeGrid.js

```diff
@@ -377,6 +395,17 @@ export default class TimeGrid extends Component {
     }
   }
 
+  measureAllDayRowWidth() {
+    let { allDayRowWidth } = this.state || {};
```

Comment:
Maybe add `allDayRowWidth` to `this.state` in `constructor`? Even though it doesn't have a meaningful initial value, but makes it clearer what are the internal states are in this component.